go/private/extensions.bzl

```diff
@@ -39,6 +40,12 @@ _host_tag = tag_class(
     },
 )
 
+_nogo_tag = tag_class(
+    attrs = {
+        "nogo": attr.string(mandatory = True),
```

Comment:
The "nogo" attribute stutters with the tag class name. Could we either use something different for the attribute name here such as "rule", or perhaps generalize the tag class name to something like "tools"?